backend/aquarium.py

```diff
@@ -55,11 +56,13 @@ def add_aquarium():
         "height": height,
         "width": width,
         "length": length,
-        "image": "https://alerybka.pl/wp-content/uploads/2021/09/dojrzale-akwarium.jpeg",
+        "image": image,
         "heater_id": heater_id,
         "lamp_id": lamp_id,
         "pump_id": pump_id,
         "filter_id": filter_id,
+        "water_min": [{"KH": "", "GH": "", "pH": "", "NO2": "", "NO3": ""}],
+        "water_max": [{"KH": "", "GH": "", "pH": "", "NO2": "", "NO3": ""}],
```

Comment:
I think that zero as numbers should be better empty value, but I can also make this into frontend